src/hooks/useCount.js

```diff
@@ -1,15 +1,17 @@
 import { useState } from "react";
 
-export default function useCount() {
+export default function useCount(itemId, updateItemQuantity) {
   const [count, setCount] = useState(0);
 
   function increment() {
     setCount(count + 1);
+    updateItemQuantity(itemId, count + 1);
```

Comment:
I think it would be a good idea to have a max amount of items I can order. I just ordered 202 dishes of Mandu. :)

Reply:
hahahaahaha!!! :D okay got it